specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/examples/agreementByName.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2023-04-01",
+    "billingAccountName": "10000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31",
+    "agreementName": "ABC123"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Billing/billingAccounts/10000000-0000-0000-0000-000000000000:00000000-0000-0000-0000-000000000000_2019-05-31/agreements/ABC123",
+        "name": "ABC123",
+        "properties": {
+          "acceptanceMode": "ClickToAccept",
+          "agreementLink": "https://agreementuri1.com",
```

Comment:
Let's use safe urls (e.g. contoso.com), just in case one day a malicious user grabs this domain and uses it for phishing purposes.